### PR TITLE
Frontend setup + Implement Navbar

### PR DESCRIPTION
**Pull request by CharleneKwok**

### CODE REVIEW

File: ui/public/index.html [hunk 1]
```diff
@@ -24,20 +23,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Weather Telemetry</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (lhjt) [inline ui/public/index.html:30]: Is there a reason why you don't want this tag?

File: ui/src/index.tsx [hunk 2]
```diff
@@ -2,16 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.render(
-  <React.StrictMode>
```
Comment (lhjt) [inline ui/src/index.tsx:8]: Why were the `StrictMode` tags removed?

File: ui/public/index.html (hunk 1 above)
Comment (CharleneKwok) [reply ui/public/index.html:30]: Oh I removed it accidentally. I would bring it back.

File: ui/src/index.tsx (hunk 2 above)
Comment (CharleneKwok) [reply ui/src/index.tsx:8]: StrictMode would cause the component to run twice and it is just for development. There is no effect to leave that out

**Comment by lhjt** [comment]
image
Would it be possible to have this border removed when this is selected?

**Comment by CharleneKwok** [comment]
> image Would it be possible to have this border removed when this is selected?

Yeah, Removed!

**Pull request merged**